lighthouse-extension/app/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "__MSG_appName__",
-  "version": "1.6.0",
+  "version": "2.0.0.1",
```

Comment:
CWS doesnt allow text in the version names, so i can't do "alpha". I was hesitant to number it as 1.x, though.

what do you suggest? 

i can bring it back to 1.99.x but i'll have to get a new extension ID. (CWS wont let you go backwards)